### PR TITLE
Update collection version on tag publish

### PR DESCRIPTION
**Pull request by mnasiadka**

### CODE REVIEW

File: .github/workflows/publish-collection.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ jobs:
         if: github.ref_type != 'tag'
       - name: Github Checkout 🛎
         uses: actions/checkout@v4
-      - name: Check git tag matches version in galaxy.yml
-        run: echo "${{ github.ref_name }}" | grep $(yq -r .version < galaxy.yml)
+      - name: Update galaxy.yml version to git tag
+        run: yq -i '.version = "${{ github.event.release.tag_name }}"' 'galaxy.yml'
```
Comment (markgoddard) [inline .github/workflows/publish-collection.yml:19]: Seems like a bit of a fudge? The change won't be committed.

**Comment by markgoddard** [comment]
Somehow this change is in main already: https://github.com/stackhpc/.github/commit/53db12f7846eae71df037022a34cc7d8fcb08faa

**Comment by mnasiadka** [comment]
> Somehow this change is in main already: [53db12f](https://github.com/stackhpc/.github/commit/53db12f7846eae71df037022a34cc7d8fcb08faa)

Fantastic, let me revert - have no clue how it happened.

File: .github/workflows/publish-collection.yml (hunk 1 above)
Comment (mnasiadka) [reply .github/workflows/publish-collection.yml:19]: Right, committing that is going to be a painful operation.

**Comment by mnasiadka** [comment]
> > Somehow this change is in main already: [53db12f](https://github.com/stackhpc/.github/commit/53db12f7846eae71df037022a34cc7d8fcb08faa)
> 
> Fantastic, let me revert - have no clue how it happened.

Done.

**Comment by markgoddard** [comment]
I know this is a pain - I literally just hit it :facepalm:

**Comment by markgoddard** [comment]
Wondering if we should have a GH workflow to create a release?

**Pull request closed without merging**